### PR TITLE
Use headers for file names, not code comments

### PR DESCRIPTION
**Pull request by justin808**
* Updated so it's more clear when different files are used for an example
* Clarify that the customization file is based on the NODE_ENV

### CODE REVIEW

**Comment by jakeNiemiec** [comment]
> Updated so it's more clear when different files are used for an example

I personally like the file comments since they are commonly used like that on GH. (but I don't want to nitpick)

File: docs/webpack.md [hunk 1]
```diff
@@ -155,9 +159,8 @@ fileLoader.exclude = /\.(svg)$/i
 
 ### Url Loader
 
+#### config/webpack/loaders/url.js
 ```js
-// config/webpack/loaders/url.js
```
Comment (jakeNiemiec) [inline docs/webpack.md]: This might be a mistake, but I may have just never used webpacker like this.

Comment (justin808) [reply docs/webpack.md:159]: I don't understand this comment. The PR adds headers.

**Comment by justin808** [comment]
@gauravtiwari any comments?

**Comment by justin808** [comment]
Closing in favor of #2748.

**Pull request closed without merging**